src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -183,7 +183,7 @@ internal class CommandLineParameterParser
             "file",
```

Comment:
@rjmholt, your last commit had 6 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

ConsoleHost unit tests.-Login pwsh switch.Accepts the -l switch for -Login and behaves correctly
```powershell
Expected exactly 5701bc7f-2198-4485-97d5-e3cbeca0856e, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 307
307:             $result | Should -BeExactly $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -L switch for -Login and behaves correctly
```powershell
Expected exactly 5701bc7f-2198-4485-97d5-e3cbeca0856e, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 307
307:             $result | Should -BeExactly $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -login switch for -Login and behaves correctly
```powershell
Expected exactly 5701bc7f-2198-4485-97d5-e3cbeca0856e, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 307
307:             $result | Should -BeExactly $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -Login switch for -Login and behaves correctly
```powershell
Expected exactly 5701bc7f-2198-4485-97d5-e3cbeca0856e, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 307
307:             $result | Should -BeExactly $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -LOGIN switch for -Login and behaves correctly
```powershell
Expected exactly 5701bc7f-2198-4485-97d5-e3cbeca0856e, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 307
307:             $result | Should -BeExactly $guid
```